app/screens/receive-bitcoin-screen/receive-btc.tsx

```diff
@@ -245,46 +245,56 @@ const ReceiveBtc = () => {
             hasAmount: false,
             receivingWallet: WalletCurrency.Btc,
           })
-          const {
-            data: {
-              lnNoAmountInvoiceCreate: { invoice, errors },
-            },
-          } = await lnNoAmountInvoiceCreate({
+          const { data } = await lnNoAmountInvoiceCreate({
             variables: { input: { walletId, memo } },
           })
+
+          if (!data) {
+            throw new Error("No data returned from lnNoAmountInvoiceCreate")
```

Comment:
Same question here - is this caught or does it bubble up to the error screen?

Reply:
Caught in 294

Reply:
Right enough.  Should have looked at the whole file I thought this was part of the render method 😅